diff -U 3 -H -d -r -N -- ffDiaporama-orig/src/ffDiaporama/DlgImage/wgt_QEditVideo/wgt_QEditVideo.h ffDiaporama-patched/src/ffDiaporama/DlgImage/wgt_QEditVideo/wgt_QEditVideo.h
--- ffDiaporama-orig/src/ffDiaporama/DlgImage/wgt_QEditVideo/wgt_QEditVideo.h	2014-05-04 08:04:04.000000000 +0200
+++ ffDiaporama-patched/src/ffDiaporama/DlgImage/wgt_QEditVideo/wgt_QEditVideo.h	2016-08-09 15:30:04.969338626 +0200
@@ -27,11 +27,7 @@
 #include "engine/_Diaporama.h"
 #include "CustomCtrl/QCustomRuler.h"
 
-#if QT_VERSION >= 0x050000
 #include <QtMultimedia/QAudioOutput>
-#else
-#include <QAudioOutput>
-#endif
 
 namespace Ui {
     class wgt_QEditVideo;
diff -U 3 -H -d -r -N -- ffDiaporama-orig/src/ffDiaporama/DlgMusic/DlgEditMusic.h ffDiaporama-patched/src/ffDiaporama/DlgMusic/DlgEditMusic.h
--- ffDiaporama-orig/src/ffDiaporama/DlgMusic/DlgEditMusic.h	2014-05-04 08:05:19.000000000 +0200
+++ ffDiaporama-patched/src/ffDiaporama/DlgMusic/DlgEditMusic.h	2016-08-09 15:32:02.465341307 +0200
@@ -26,11 +26,7 @@
 #include "engine/_Diaporama.h"
 #include "CustomCtrl/QCustomRuler.h"
 
-#if QT_VERSION >= 0x050000
 #include <QtMultimedia/QAudioOutput>
-#else
-#include <QAudioOutput>
-#endif
 
 namespace Ui {
     class DlgEditMusic;
diff -U 3 -H -d -r -N -- ffDiaporama-orig/src/ffDiaporama/ffDiaporama.pro ffDiaporama-patched/src/ffDiaporama/ffDiaporama.pro
--- ffDiaporama-orig/src/ffDiaporama/ffDiaporama.pro	2014-05-04 10:33:56.000000000 +0200
+++ ffDiaporama-patched/src/ffDiaporama/ffDiaporama.pro	2016-08-09 15:44:12.275357960 +0200
@@ -38,7 +38,7 @@
     }
 }
 
-QT          += core gui xml network svg sql
+QT          += core gui xml network svg sql multimedia
 QMAKE_STRIP  = echo
 APPFOLDER    = ffDiaporama
 TARGET       = ffDiaporama
diff -U 3 -H -d -r -N -- ffDiaporama-orig/src/ffDiaporama/wgt_QVideoPlayer/wgt_QVideoPlayer.cpp ffDiaporama-patched/src/ffDiaporama/wgt_QVideoPlayer/wgt_QVideoPlayer.cpp
--- ffDiaporama-orig/src/ffDiaporama/wgt_QVideoPlayer/wgt_QVideoPlayer.cpp	2014-05-10 09:22:20.000000000 +0200
+++ ffDiaporama-patched/src/ffDiaporama/wgt_QVideoPlayer/wgt_QVideoPlayer.cpp	2016-08-09 15:36:48.529347835 +0200
@@ -21,7 +21,7 @@
 #include "wgt_QVideoPlayer.h"
 #include "MainWindow/mainwindow.h"
 #include "ui_wgt_QVideoPlayer.h"
-#include <QAudioFormat>
+#include <QtMultimedia/QAudioFormat>
 
 #define ICON_PLAYERPLAY                     ":/img/player_play.png"                 // FileName of play icon
 #define ICON_PLAYERPAUSE                    ":/img/player_pause.png"                // FileName of pause icon
diff -U 3 -H -d -r -N -- ffDiaporama-orig/src/ffDiaporama/wgt_QVideoPlayer/wgt_QVideoPlayer.h ffDiaporama-patched/src/ffDiaporama/wgt_QVideoPlayer/wgt_QVideoPlayer.h
--- ffDiaporama-orig/src/ffDiaporama/wgt_QVideoPlayer/wgt_QVideoPlayer.h	2014-05-04 07:58:48.000000000 +0200
+++ ffDiaporama-patched/src/ffDiaporama/wgt_QVideoPlayer/wgt_QVideoPlayer.h	2016-08-09 15:25:39.455332568 +0200
@@ -27,11 +27,7 @@
 #include "CustomCtrl/QCustomRuler.h"
 #include "CustomCtrl/QMovieLabel.h"
 
-#if QT_VERSION >= 0x050000
 #include <QtMultimedia/QAudioOutput>
-#else
-#include <QAudioOutput>
-#endif
 
 namespace Ui {
     class wgt_QVideoPlayer;
