diff -U 3 -H -d -r -N -- cinelerra-goodguy/cinelerra-5.1/Makefile.am cinelerra-goodguy-patched/cinelerra-5.1/Makefile.am
--- cinelerra-goodguy/cinelerra-5.1/Makefile.am	2016-06-04 04:04:40.025111851 +0200
+++ cinelerra-goodguy-patched/cinelerra-5.1/Makefile.am	2016-06-04 15:11:53.906279027 +0200
@@ -50,7 +50,6 @@
 		COPYING README msg.txt ffmpeg doc models
 	cd bin/locale; $(inst_sh) "$(DESTDIR)$(localedir)" .
 	cd bin/plugins; $(inst_sh) "$(DESTDIR)$(WANT_PLUGIN_DIR)" .
-	cd bin/ladspa; $(inst_sh) "$(DESTDIR)$(WANT_LADSPA_DIR)" .
 	cd bin/applications; $(inst_sh) "$(DESTDIR)$(datadir)/applications" .
 	cd bin/pixmaps; $(inst_sh) "$(DESTDIR)$(datadir)/pixmaps" .
 
diff -U 3 -H -d -r -N -- cinelerra-goodguy/cinelerra-5.1/configure.ac cinelerra-goodguy-patched/cinelerra-5.1/configure.ac
--- cinelerra-goodguy/cinelerra-5.1/configure.ac	2016-06-04 04:04:40.217111855 +0200
+++ cinelerra-goodguy-patched/cinelerra-5.1/configure.ac	2016-06-04 14:40:12.859201837 +0200
@@ -25,7 +25,6 @@
 CHECK_WITH([jobs],[parallel build jobs],[JOBS],[auto])
 CHECK_WITH([exec-name],[binary executable name],[CIN],[cin])
 CHECK_WITH([single-user],[to install cin in bin],[CINBIN_BUILD],[no])
-CHECK_WITH([ladspa-build],[build ladspa library],[LADSPA_BUILD],[yes])
 CHECK_WITH([cinlib],[cinelerra library path],[CINLIB_DIR],[auto])
 CHECK_WITH([cindat],[cinelerra share path],[CINDAT_DIR],[auto])
 CHECK_WITH([plugin-dir],[plugin install dir],[PLUGIN_DIR],[auto])
@@ -186,8 +185,6 @@
     IlmThread/.libs/libIlmThread.a ],
   [ Iex Half Imath config IlmThread ])
 
-PKG_DEF([ladspa], [ladspa-05182015], [], [])
-
 PKG_3RD([lame],[auto],
   [lame-3.99.5],
   [ libmp3lame/.libs/libmp3lame.a \
@@ -610,7 +607,6 @@
 PKG_PROVIDE([flac])
 PKG_PROVIDE([giflib])
 PKG_PROVIDE([ilmbase])
-PKG_PROVIDE([ladspa],[$WANT_LADSPA_BUILD])
 PKG_PROVIDE([lame])
 PKG_PROVIDE([libavc1394],[$WANT_FIREWIRE])
 PKG_PROVIDE([libraw1394],[$WANT_FIREWIRE])
@@ -663,7 +659,6 @@
 echo ""
 echo "  using: single-user  = $WANT_CINBIN_BUILD"
 echo "  using: static-build = $WANT_STATIC_BUILD"
-echo "  using: ladspa-build = $WANT_LADSPA_BUILD"
 echo ""
 
 # build extras
diff -U 3 -H -d -r -N -- cinelerra-goodguy/cinelerra-5.1/thirdparty/Makefile cinelerra-goodguy-patched/cinelerra-5.1/thirdparty/Makefile
--- cinelerra-goodguy/cinelerra-5.1/thirdparty/Makefile	2016-06-04 04:04:41.220111878 +0200
+++ cinelerra-goodguy-patched/cinelerra-5.1/thirdparty/Makefile	2016-06-04 14:40:57.340203643 +0200
@@ -173,7 +173,6 @@
 mjpegtools.cflags?="$(call inc_path,libjpeg) $(call ld_path,libjpeg,.libs)"
 mjpegtools.mak_params?=; ln -s . $(call bld_path,mjpegtools,utils)/mjpegtools 
 mjpegtools.cfg_params?= --enable-shared=no
-ladspa.mak_params?=; $(MAKE) -C ladspa* install DESTDIR=$(call bld_path,ladspa)
 libavc1394.cfg_vars?=PKG_CONFIG_PATH=$(call bld_path,libraw1394)
 libavc1394.cflags?="$(call inc_path,libraw1394)"
 libavc1394.ldflags?="$(call ld_path,libraw1394,src/.libs)"
@@ -239,7 +238,6 @@
 $(call rules,$(call std-build,flac,libogg))
 $(call rules,$(call std-build,giflib))
 $(call rules,$(call std-build,ilmbase))
-$(call rules,$(call std-build,ladspa))
 $(call rules,$(call std-build,lame))
 $(call rules,$(call std-build,libavc1394))
 $(call rules,$(call std-build,libdv))
