diff -U 3 -H -d -r -N -- a/cinelerra-5.1/thirdparty/Makefile b/cinelerra-5.1/thirdparty/Makefile
--- a/cinelerra-5.1/thirdparty/Makefile	2016-04-19 07:46:12.586300494 +0200
+++ b/cinelerra-5.1/thirdparty/Makefile	2016-04-19 07:47:57.226304743 +0200
@@ -102,6 +102,7 @@
 ffmpeg.cfg_params= \
 	--enable-pthreads --enable-gpl \
 	--disable-ffserver --disable-ffplay \
+	--disable-vaapi \
 	--enable-libfaac \
 	--enable-libfdk-aac \
 	--enable-libtwolame \
@@ -159,7 +160,6 @@
 mjpegtools.cflags="$(call inc_path,libjpeg) $(call ld_path,libjpeg,.libs)"
 mjpegtools.mak_params=; ln -s . $(call bld_path,mjpegtools,utils)/mjpegtools 
 mjpegtools.cfg_params= --enable-shared=no
-ladspa.mak_params=; $(MAKE) -C ladspa* install DESTDIR=$(call bld_path,ladspa)
 libavc1394.cfg_vars=PKG_CONFIG_PATH=$(bld_path libraw1394)
 libavc1394.cflags="$(inc_path libraw1394)"
 libavc1394.ldflags="$(ld_path libraw1394,src/.libs)"
@@ -221,7 +221,6 @@
 $(call rules,$(call std-build,flac,libogg))
 $(call rules,$(call std-build,giflib))
 $(call rules,$(call std-build,ilmbase))
-$(call rules,$(call std-build,ladspa))
 $(call rules,$(call std-build,lame))
 $(call rules,$(call std-build,libavc1394))
 $(call rules,$(call std-build,libdv))
diff -U 3 -H -d -r -N -- a/cinelerra-5.1/thirdparty/configure b/cinelerra-5.1/thirdparty/configure
--- a/cinelerra-5.1/thirdparty/configure	2016-04-19 07:46:12.586300494 +0200
+++ b/cinelerra-5.1/thirdparty/configure	2016-04-19 07:48:31.422306131 +0200
@@ -133,8 +133,6 @@
 	Half/.libs/libHalf.a \
 	Imath/.libs/libImath.a \
 	IlmThread/.libs/libIlmThread.a
-inc_ladspa=""
-add_library ladspa
 inc_lame="include"
 add_library lame \
 	libmp3lame/.libs/libmp3lame.a \
