diff -u -r VirtualBox-4.3.20.orig/src/VBox/Main/webservice/Makefile.kmk VirtualBox-4.3.20/src/VBox/Main/webservice/Makefile.kmk
--- VirtualBox-4.3.20.orig/src/VBox/Main/webservice/Makefile.kmk	2014-11-21 16:22:38.000000000 +0100
+++ VirtualBox-4.3.20/src/VBox/Main/webservice/Makefile.kmk	2014-12-10 20:46:37.462326942 +0100
@@ -159,7 +159,6 @@
 RECOMPILE_ON_MAKEFILE_CURRENT := $(MAKEFILE_CURRENT)
 
 PATH_TARGET_SOAPDEMOXML       := $(VBOXWEB_OUT_DIR)/demo_soapxml
-PATH_TARGET_SOAPDEMOHEADERS   := $(VBOXWEB_OUT_DIR)/demo_headers
 PATH_TARGET_SOAPDEMONSMAPS    := $(VBOXWEB_OUT_DIR)/demo_namespacemaps
 PATH_TARGET_WEBTEST           := $(VBOXWEB_OUT_DIR)/webtest
 
@@ -512,8 +511,6 @@
 	$(VBOXWEB_OUT_DIR)/gsoap_copy_all_ts \
 	$(wildcard $(PATH_TARGET_SOAPDEMOXML)/*) \
 	$(PATH_TARGET_SOAPDEMOXML)/dummy_file \
-	$(wildcard $(PATH_TARGET_SOAPDEMOHEADERS)/*) \
-	$(PATH_TARGET_SOAPDEMOHEADERS)/dummy_file \
 	$(wildcard $(PATH_TARGET_SOAPDEMONSMAPS)/*) \
 	$(PATH_TARGET_SOAPDEMONSMAPS)/dummy_file
